test/examples/src/ts/UpdateDoc.ts

```diff
@@ -35,6 +35,15 @@ const getDocParams: CloudantV1.GetDocumentParams = {
   db: exampleDbName,
 };
 
+// Note: for byte response (Output Stream) use:
+// const getdocAsStreamParam: CloudantV1.GetDocumentAsStreamParams = {
+//   docId: 'example',
+//   db: exampleDbName,
+// };
+// client
+//   .getDocumentAsStream(getdocAsStreamParam)
+//   .then((outputStream) => {});
```

Comment:
What do you think of this:
```suggestion
// const request = client. getDocumentAsStream(getdocAsStreamParam);

const request = client. getDocument(getDocParams);

request
  .then((response)) => {
  
      // Note: for byte response ...
      // here would be the same modifications as in Document case below
      // but for the byte response
  
      // using OrderDocument on getDocument result:
      const document: OrderDocument = docResult.result;
      // Add Bob Smith's address to the document
      document.address = '19 Front Street, Darlington, DL5 1TY';
      // Remove the joined property from document object
      delete document.joined;
      
      ...
  }
```

Reply:
Nm, I was thinking it over, I guess the current solution will be better.

Reply:
Okay. 🙂